localization/deviation_estimation_tools/deviation_estimator/launch/deviation_estimator.launch.xml

```diff
@@ -1,13 +1,13 @@
 <launch>
   <arg name="use_sim_time" default="true" description="use_sim_time"/>
   <arg name="dt_design" default="10.0" description="[s]"/>
-  <arg name="dx_design" default="30.0" description="[m]"/>
+  <arg name="dx_design" default="100.0" description="[m]"/>
```

Comment:
I change the default value of `dx_design` (= expected distance that the vehicle runs without LiDAR localization) to 100[m] to take the safe side.